script.js

```diff
@@ -0,0 +1,40 @@
+var rooot = document.getElementById('root');

```

Comment:
Please, change  'var' into modern js -  'let' or 'const'

Reply:
'rooot' or 'root'? 
You can add extension if you use Visual Studio for checking words speller - 'Code Spell Checker'.

Reply:
I made the necessary changes you mentioned, in new branch.
Thank you!